.github/workflows/test.yml

```diff
@@ -10,58 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tarantool:
-          - "1.10"
-          - "2.2"
-          - "2.3"
-          - "2.4"
-          - "2.5"
-          - "2.6"
-          - "2.7"
-          - "2.8"
-        cartridge:
-          - ""
-          - "1.2.0"
-          - "2.1.2"
-          - "2.4.0"
-          - "2.5.1"
-          - "2.6.0"
-          - "2.7.9"
         include:
-          # 1.10
-          - tarantool: "1.10"
-            cartridge: "2.8.6"
-          - tarantool: "1.10"
-            cartridge: "2.9.0"
-          - tarantool: "1.10"
-            cartridge: "2.10.0"
-          - tarantool: "1.10"
-            cartridge: "2.10.0"
-          - tarantool: "1.10"
-            cartridge: "2.11.0"
-          - tarantool: "1.10"
-            cartridge: "2.12.4"
-          - tarantool: "1.10"
-            cartridge: "2.13.0"
-          # 2.10
-          - tarantool: "2.10"
-            cartridge: "2.7.9"
-          - tarantool: "2.10"
-            cartridge: "2.8.6"
-          - tarantool: "2.10"
-            cartridge: "2.9.0"
-          - tarantool: "2.10"
-            cartridge: "2.10.0"
-          - tarantool: "2.10"
-            cartridge: "2.10.0"
-          - tarantool: "2.10"
-            cartridge: "2.11.0"
-          - tarantool: "2.10"
-            cartridge: "2.12.4"
-          - tarantool: "2.10"
-            cartridge: "2.13.0"
-          - tarantool: "2.10"
-            cartridge: ""
```

Comment:
I would add the latest cartridge versions here, but it's up to you

Reply:
For 2.10 there is no point in it, since there are no its packages for Ubuntu noble.